herring/puzzles/models.py

```diff
@@ -58,6 +59,7 @@ class Json:
 
 class Puzzle(models.Model,JSONMixin):
     # class for all puzzles, including metas
+    hunt_id = models.IntegerField(default=settings.HERRING_HUNT_ID)  # which hunt is this for? Will turn into a foreign key later.
```

Comment:
One additional consequence of keeping `Puzzle` rows around forever is that `ChannelParticipation` rows also will remain forever (I had set them up to get cascade-deleted when their corresponding puzzle is deleted). Low-priority, but we may want to run a cleanup of that table yearly, since I assume that's not data we care about retaining indefinitely, and I don't think there's a cost-free way to index that table such that old rows don't slow down the one query we do against it. On the other hand, that one query is currently debounced to once every five seconds, so performance issues with it are unlikely to be a problem anytime soon if we don't do anything.